src/types/metadata.ts

```diff
@@ -72,6 +72,11 @@ export type Metadata = {
  */
 export type TypeId = keyof Metadata;
 
+export interface Localization {
+  name: string;
+  description: string;
```

Comment:
Should we make them optional since we will probably be reusing this interface for other schemas? For example, I see token metadata just have description and no name, but strategies does have both

Reply:
Are we sure? I think name is coming in both. I'll check to make sure